app/models/user.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import annotations
+from dataclasses import dataclass
+import logging
+from typing import Optional
+
+from app.config import Config
+
+INSERT_SQL = '''
+    INSERT INTO "users"
+        ("login", "bind_token", "email2", "phone", "telegram", "otp", "reset_token")
+    VALUES (?, ?, ?, ?, ?, ?, ?)
+'''
+UPDATE_SQL = '''
+    UPDATE "users"
+    SET "login" = ?,
+        "bind_token" = ?,
+        "email2" = ?,
+        "phone" = ?,
+        "telegram" = ?,
+        "otp" = ?,
+        "reset_token" = ?
+    WHERE "id" = ?
+'''
+FETCH_SQL = 'SELECT "id", "login", "bind_token", "email2", "phone", "telegram", "otp", "reset_token" FROM "users"'
+FETCH_BY_LOGIN_SQL = '''
+    SELECT "id", "bind_token", "email2", "phone", "telegram", "otp", "reset_token"
+    FROM "users"
+    WHERE "login" = ?
+'''
+FETCH_BY_TELEGRAM_SQL = '''
+    SELECT "login"
+    FROM "users"
+    WHERE "telegram" = ?
+'''
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class User:
+    id: Optional[int] = None
+    login: Optional[str] = None
+    bind_token: Optional[str] = None
+    email2: Optional[str] = None
+    phone: Optional[int] = None
+    telegram: Optional[int] = None
+    otp: Optional[int] = None
+    reset_token: Optional[int] = None
+
+    # only from LDAP
+    name: Optional[str] = None
+
+    def save(self):
```

Comment:
This type of methods usually return object they create or throw exception if failed